argoexec/rockcraft.yaml

```diff
@@ -65,30 +52,25 @@ parts:
       - mailcap
       - procps
       - tar
-      - libcap2-bin
       - libcap2-dev
       - jq
-      - zip
     stage-packages:
       - git
       - libcap2
       - jq
       - tar
       - gzip
     override-build: |
-      # builder stage https://github.com/argoproj/argo-workflows/blob/v3.3.10/Dockerfile#L36
+      # builder stage https://github.com/argoproj/argo-workflows/blob/v3.4.16/Dockerfile#L36
       go mod download
 
       # reset LDFLAGS because Go has different format, i.e. '-L' is undefined in Go
       LDFLAGS=""
 
-      # argoexec-build stage https://github.com/argoproj/argo-workflows/blob/v3.3.10/Dockerfile#L82
-      cat .dockerignore >> .gitignore
-      git status --porcelain | cut -c4- | xargs git update-index --skip-worktree
+      # argoexec-build stage https://github.com/argoproj/argo-workflows/blob/v3.4.16/Dockerfile#L82
       make dist/argoexec
-      setcap CAP_SYS_PTRACE,CAP_SYS_CHROOT+ei dist/argoexec
 
-      # argoexec stage https://github.com/argoproj/argo-workflows/blob/v3.3.10/Dockerfile#L132
+      # argoexec stage https://github.com/argoproj/argo-workflows/blob/v3.4.16/Dockerfile#L132
```

Comment:
I think the comment line refference is out of date

Reply:
See https://github.com/canonical/argo-workflows-rocks/pull/21#discussion_r1627130205